src/task.js

```diff
@@ -21,10 +21,9 @@ export default class Task {
   }
 
   removeMultiple = (array) => {
-    for (let i = array.length - 1; i >= 0; i -= 1) {
```

Comment:
- ### This inline comment is intended for README file
 - [ ] Kindly add the required command for running your project `npm start` 
 - [ ] Your live link doesn't send to your page live link, either add it or if you don't have a live link just add N/A or explain that you will add it later.